build/build.py

```
@@ -48,7 +48,10 @@ def is_windows():
 
 
 def shell(cmd):
-  output = subprocess.check_output(cmd)
+  try:
+    output = subprocess.check_output(cmd)
+  except subprocess.CalledProcessError as e:
+    print(e.output)
```

I think we should reraise the exception here?

---

Done

---

I think to reraise you can just write `raise` with no `e`.

---

done